### PR TITLE
[IOS-573] Search users bots groups view

### PR DESCRIPTION
**Pull request by marijnschilling**
Maybe its just done like this actually, the dataSource can be added easily ✌️ 

I added it to the NewGroupViewController and the FindUserBotsGroups one:
search

### CODE REVIEW

File: Toshi/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -353,6 +353,11 @@
 "chat_payment_recieved" = "Payment received";
 "chat_payment_sent" = "Payment sent";
 
+/* Search Chat */
+"search_chat_users_tab_name" = "Users";
+"search_chat_groups_tab_name" = "Bots";
```
Comment (yuliaveres) [inline Toshi/Resources/Base.lproj/Localizable.strings]: groups

File: Toshi/Resources/Base.lproj/Localizable.strings [hunk 2]
```diff
@@ -353,6 +353,11 @@
...
+"search_chat_bots_tab_name" = "Groups";
```
Comment (yuliaveres) [inline Toshi/Resources/Base.lproj/Localizable.strings]: bots

Comment (marijnschilling) [reply Toshi/Resources/Base.lproj/Localizable.strings]: oops 🙈

**Comment by marijnschilling** [comment]
img_10ce27032c77-1
All better now 😅

**Comment by yuliaveres** [comment]
All good 👍

**Pull request merged**